src/main.rs

```diff
@@ -41,9 +42,11 @@ fn real_main() -> Result<i32, Box<dyn std::error::Error>> {
         select_response(ResponseType::Negative)
     };
 
+    let stylize = std::io::stderr().is_terminal();
     match response {
```

Comment:
IMO this is better done by matching over a tuple of `(response, stylize)`. Matter of taste, but that lets you rely on the exhaustiveness checks on `match`. (Since as-is, the match lines are order-dependent).

Like

```rust
    match (response, stylize) {
        (Ok(resp), true) => eprintln
        (Err(resp), true) => eprintln
        (Ok(resp) | Err(resp), false) => eprintln
    }
```

Reply:
(oops, this PR got merge conflicted so might as well do this cleanup to before landing)

Reply:
Updated